### PR TITLE
Wait for a second between each check

### PR DESCRIPTION
**Pull request by jrjohnson**

### CODE REVIEW

File: src/get-survey-results.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ async function waitForBuildToComplete(token, dataCenter, progressId) {
   if (data.result.percentComplete === 100) {
     return data.result.file;
   }
-
+  //wait for a second
+  Atomics.wait(new Int32Array(new SharedArrayBuffer(4)), 0, 0, 1000);
```
Comment (Trott) [inline src/get-survey-results.js]: I think `Atomics.wait()` will block the event loop, but I'm not sure if that's a problem in this context.

File: src/get-survey-results.js [hunk 2]
```diff
@@ -33,7 +33,8 @@ async function waitForBuildToComplete(token, dataCenter, progressId) {
...
   return waitForBuildToComplete(token, dataCenter, progressId);
```
Comment (Trott) [inline src/get-survey-results.js]: Maybe this instead?:

```suggestion
  // Try again after one second.
  return await new Promise(() => { setTimeout(waitForBuildToComplete, 1000, token, dataCenter, progressId); });
```

 ^^^^^untested

Comment (jrjohnson) [reply src/get-survey-results.js]: I'm just eyeballing it, but I don't think that promise would ever resolve. I don't know, I can never reason about `setTimeout` effectivly. I can switch it up to something like https://stackoverflow.com/a/41957152/796999 instead. I agree, I don't know anything about Atomics.wait. Total copy paste job.

**Comment by jrjohnson** [comment]
Thanks @Trott, I pulled out a `sleep` function and am using that now.

File: src/get-survey-results.js (hunk 2 above)
Comment (Trott) [reply src/get-survey-results.js]: Whoops, you're right.

Let me check that I'm not completely wrong about Atomics....

**Comment by Trott** [comment]
> Thanks @Trott, I pulled out a `sleep` function and am using that now.

Looks like Atomics.wait() does block the event loop (at least as long as I did my testing correctly) and this does not, so 👍.

**Pull request merged**